ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigBuilder.java

```diff
@@ -19,19 +19,20 @@
 import static tech.pegasys.teku.spec.constants.NetworkConstants.DEFAULT_SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY;
 
 import java.math.BigInteger;
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.function.Consumer;
 import org.apache.tuweni.bytes.Bytes32;
 import org.apache.tuweni.units.bigints.UInt256;
 import tech.pegasys.teku.ethereum.execution.types.Eth1Address;
 import tech.pegasys.teku.infrastructure.bytes.Bytes4;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 
+@SuppressWarnings({"UnusedReturnValue", "unused"})
 public class SpecConfigBuilder {
+
+  // Placeholder version explicitly doesn't match MainNet (or any other known testnet)
+  private static final Bytes4 PLACEHOLDER_FORK_VERSION = Bytes4.fromHexString("0x99999999");
```

Comment:
want to set this to `0xffffffff`?

Reply:
eh. It's entirely arbitrary and never actually used. Just making it different to MainNet for safety really.